SIG.md

```diff
@@ -0,0 +1,107 @@
+# Cilium SIGS
+
+The Cilium project has Special Interest Groups, or SIGs, with a common purpose of advancing the project with respect to a specific topic, such as network policy or documentation. Our goal is to enable a distributed decision structure and code ownership, as well as providing focused forums for getting work done, making decisions, and onboarding new contributors. 
+
+Areas covered by SIGs may be vertically focused on particular components or functions, cross-cutting/horizontal, spanning many/all functional areas of the project, or in support of the project itself. Examples:
+
+- Policy
+- Scalability
+- Docs
+
+Each SIG must have a charter that specifies its scope (topics, subsystems, code repos, and/or directories), responsibilities, areas of authority, and basic governance. See the SIG charter process for details on how charters are managed. SIGs are relatively free to customize or change how they operate, within some broad guidelines and constraints imposed by cross-project processes (e.g., the release process) and assets (e.g., the cilium repo).
```

Comment:
Do we want SIGs to have their own governance at this stage? Also unsure about "areas of authority" at this point (whether this isn't already covered by the authority defined by project roles)

Reply:
For basic governance, I meant who the leaders are, but I could also see how that could be confusing.

Reply:
We might also want to flesh out the relationship between project roles (reviewer, committer) and SIGs. For example, I think one of the goals was that people who are aspiring to be reviewers can ask to build up expertise by getting assigned PRs to review within the SIG.